### PR TITLE
Auth login/logout

### PR DESCRIPTION
**Pull request by sameep-s**
Added Login/logout/signup.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ import NotificationPage from './NotificationPage/NotificationPage';
 import BookmarksPage from './BookMarksPage/BookmarksPage';
```
Comment (SaurabhChawda) [inline src/pages/index.js:5]: You can create Index File for import and export.

**Review by SaurabhChawda — APPROVED** [review]
Test Login Button is not working.

File: src/pages/index.js (hunk 1 above)
Comment (sameep-s) [reply src/pages/index.js:5]: sure, thanks for suggestion.

File: src/features/authSlice.js [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+            email: email,
+            username: username,
+            password: password,
+            confirmPassword: confirmPassword,
+            firstName: firstName,
+            lastName: lastName,
+            age: age
+        });
+
+        console.log(data);
```
Comment (iarjunbharti) [inline src/features/authSlice.js:32]: remove logs

**Review by iarjunbharti — APPROVED** [review]
just added a minor suggestion. everything else is fine!

File: src/features/authSlice.js (hunk 2 above)
Comment (sameep-s) [reply src/features/authSlice.js:32]: will do

**Pull request merged**